src/mudsys/save.177

```diff
@@ -319,7 +319,7 @@ IFN ITS,[
 ]
 
 ; NOW CYCLE THROUGH CHANNELS
-	MOVE	C,[-N.CHNS*2,,CHNL1]	; POINT TO REAL CHANNELS SLOTS
```

Comment:
Does multiplying a global work with the MIDAS "323" + STINK "2" combo?  If so, maybe this can be dropped.

Reply:
Yes, it works:

```
002672:  000000000002  multiply next global reference by this                   
002673:  663013473135  reference minus global: 'N.CHNS', flags=15               
002674:  000000000000                           ;"      "                       
```

I'll drop that commit.

Reply:
I see the code that checks this is very simiilar in 323 and 458.  See MULTP1.  Better check that the generated code actually works.  If so, maybe it would be a fix to copy that code from 323 to 459!

Reply:
I've added a testcase for this to [my collection](https://github.com/atsampson/its-tests/tree/master/midas). At some point I'd like to try all of those tests on the various versions of MIDAS/STINK we have...